eslint/babel-eslint-plugin/src/rules/no-invalid-this.cjs

```diff
@@ -1,6 +1,6 @@
 const ruleComposer = require("eslint-rule-composer");
 const eslint = require("eslint");
-const eslintVersion = eslint.ESLint.version;
+const eslintVersion = eslint?.ESLint.version;
```

Comment:
```suggestion
const eslintVersion = eslint?.ESLint.version ?? "7";
```

So that this variable also contains the version with older ESLint.

Reply:
done

Reply:
sorry，" eslint?.ESLint"  is  "undefined",  So the program will still report an error because of "eslint?.ESLint.version"